Source/AutoExtern/AutoExtern.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
Is there a good place to add a description of a .NET project? I think it would be nice to easily be able to see what AutoExtern is for.

Reply:
Not that I know of; I used a README instead.

Reply:
It would be nice to have a project-wide XML doc comment, but I don't think that exists

Reply:
Wait, I had forgotten to push the README ! :) There was one in the tutorial but not one for the code.  Pushed now; https://github.com/dafny-lang/dafny/blob/9b6b22b673c983767f924f49c88436f89d0be6d0/Source/AutoExtern/README.md